From e8f36012f00beaa1391a4a504d30d5a908de978c Mon Sep 17 00:00:00 2001
From: Sandeep Sheriker M <sandeep.sheriker@microchip.com>
Date: Wed, 27 Mar 2019 14:29:14 -0700
Subject: [PATCH] configure.ac: fix build error.

updating GST_PLUGIN_LDFLAGS in configure.ac to fix build error in
version 1.14.4

Signed-off-by: Sandeep Sheriker M <sandeep.sheriker@microchip.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b325792..041e374 100644
--- a/configure.ac
+++ b/configure.ac
@@ -320,7 +320,7 @@ AC_SUBST(GST_ALL_LDFLAGS)
 
 dnl this really should only contain flags, not libs - they get added before
 dnl whatevertarget_LIBS and -L flags here affect the rest of the linking
-GST_PLUGIN_LDFLAGS="-module -avoid-version -export-symbols-regex '^[_]*gst_plugin_desc.*' $GST_ALL_LDFLAGS"
+GST_PLUGIN_LDFLAGS="-module -avoid-version $GST_ALL_LDFLAGS"
 AC_SUBST(GST_PLUGIN_LDFLAGS)
 
 dnl *** output files ***
-- 
2.7.4

